lib/internal/modules/esm/hooks.js

```diff
@@ -653,13 +654,13 @@ class HooksProxy {
           (builtinName) => {
             if (StringPrototypeStartsWith(builtinName, 'node:')) {
               builtinName = StringPrototypeSlice(builtinName, 5);
-            } else if (BuiltinModule.canBeRequiredWithoutScheme(builtinName)) {
-              throw new ERR_INVALID_ARG_VALUE('builtinName', builtinName);
+            } else if (!BuiltinModule.canBeRequiredWithoutScheme(builtinName)) {
```

Comment:
This part seems like it’s fixing a bug, not just restoring deleted tests. What was the bug here? I assume it’s covered by one of the new tests?

Reply:
You're right, the is PR adds the tests and fixes the ones that were breaking – which is kind of related, but also not the same thing. Let me see if I can find something more fitted.

Reply:
It doesn't really (or fully) revert that other commit, either, right? It's not like the mocking of `import.meta` is suddenly possible now.

Reply:
I did revert the commit, then applied some changes to make the test pass. So there's a philosophical debate to decide wether it's still a revert or not, I don't feel strongly either way, I'm happy to remove or leave the revert part of the commit message if you have an opinion.

Reply:
I just noticed that the PR branch contains some unrelated commit, so I'd need to rebase and force-push anyway, I can reword the commit message at that occasion if you want.

Reply:
I did remove the revert mention, instead I'm adding `Refs:` to the PR that removed the tests. Marking this discussion as resolved, let me know if that doesn't work for you.